tests/unit/test_scraper.py

```
@@ -21,830 +21,339 @@
+
+def test_parse_response(scraper):
+    soup = BeautifulSoup(MOCK_HTML, 'html.parser')
+    result = scraper.parse_response(soup)
     assert len(result) > 0
-    assert all(isinstance(item, dict) for item in result)
-    assert all('model' in item and 'price' in item for item in result)
-    mock_scraper.session.get.assert_called_once_with(url, timeout=mock_scraper.config.get('timeout', 30))
+    assert all(isinstance(item, PriceData) for item in result)
+
+def test_measure_request(scraper):
+    start_time = datetime.now()
```

Ruff (0.8.2)

Local variable `start_time` is assigned to but never used

Remove assignment to unused variable `start_time`

(F841)